### ISSUE TITLE
index.d.ts file no longer generating path?

### ISSUE THREAD

**Issue by oleersoy** [LeDDGroup/typescript-transform-paths#21]
[I changed the example](https://github.com/oleersoy/typescript-path-transform-demo) a little bit so that it consumes the sum module within the same project and does the for other modules through `index.ts`.  However now [`index.d.ts`](https://github.com/oleersoy/typescript-path-transform-demo/blob/master/target/index.d.ts) and `index.js` do not have their paths converted.  Any ideas?  I pushed the `target` directory (Compiled directory) to the github repository,

**Comment by danielpza** [comment]
It seems related with the imports/exports changes made a few commits back, I'll check it out.

**Comment by danielpza** [comment]
@oleersoy, the fix is up in v1.1.4, please upgrade. 

Also see https://github.com/LeDDGroup/typescript-transform-paths/issues/4#issuecomment-486380340 for transforming declaration files:


```diff
    "plugins": [
      { "transform": "typescript-transform-paths" },
+     { "transform": "typescript-transform-paths", "afterDeclarations": true }
    ]
```

**Comment by oleersoy** [comment]
AWESOME!  Just tested it - looking good on my end!  Thanks!
